src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -33,40 +33,23 @@
 import React from 'react';
 import '../header_logo.scss';
 import { OpenSearchDashboardsLogoDarkMode } from './opensearch_dashboards_logo_darkmode';
-
-/**
- * @param {object} logo - full logo on main screen: defaultUrl will be used in default mode; darkModeUrl will be used in dark mode
- * @param {object} mark - thumbnail logo: defaultUrl will be used in default mode; darkModeUrl will be used in dark mode
- * @param {string} applicationTitle - custom title for the application
- */
-export interface CustomLogoType {
-  darkMode: boolean;
-  logo: {
-    defaultUrl?: string;
-    darkModeUrl?: string;
-  };
-  mark: {
-    defaultUrl?: string;
-    darkModeUrl?: string;
-  };
-  applicationTitle?: string;
-}
+import { HomePageBranding } from '../../../chrome_service';
```

Comment:
better path?

Reply:
Checked and there are no shorter path for importing type Branding.